src/lib/index.ts

```diff
@@ -109,7 +103,7 @@ async function expandWithCheck ({ prefix, iri, baseIRI, types }) {
   for (const typeNamedNode of typesNamedNodes) {
     const found = dataset.match(rdf.namedNode(iri), typeTerm, typeNamedNode, graph)
     if (found.size) {
-      return [...found][0].subject.value
+      return found.toArray()[0].subject.value
```

Comment:
This should work but at runtime somehow the dataset object cannot be restructured in some cases 🤷‍♂

Reply:
For future reference: probably ts-jest's fault, transpiling ts to es5 and assuming wrongly that `xs` is the same as `[...xs]`